BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/GoogleCloudStorage/GoogleCloudStorageHealthCheck.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Text;
+using Backbone.Tooling.Extensions;
+using Google.Cloud.Storage.V1;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Backbone.BuildingBlocks.Infrastructure.Persistence.BlobStorage.GoogleCloudStorage;
+
+public class GoogleCloudStorageHealthCheck : IHealthCheck
+{
+    public GoogleCloudStorageHealthCheck(StorageClient storage, GoogleCloudStorageOptions options)
+    {
+        _storage = storage;
+        _options = options;
+    }
+
+    private readonly StorageClient _storage;
+    private readonly GoogleCloudStorageOptions _options;
+
+    private static bool? _isHealthy;
+    private const string FILE_TEXT = "healthcheck";
```

Comment:
Fields should always be placed above constructors.

Also, const fields should be placed above static fields, and static fields should be placed above instance fields.